Crash on draw_screen

##### Description:
In my project https://github.com/CameronLonsdale/MTP I have a list of edit boxes. In one edit box if I spam capital A I eventually get to the following traceback:

```
Traceback (most recent call last):
  File "/Users/clonsdale/Personal/manytime_interactive/venv/bin/mtp", line 11, in <module>
    load_entry_point('mtp', 'console_scripts', 'mtp')()
  File "/Users/clonsdale/Personal/manytime_interactive/cli.py", line 25, in main
    many_time_pad_attack(ciphertexts)
  File "/Users/clonsdale/Personal/manytime_interactive/manytime/__init__.py", line 19, in many_time_pad_attack
    interactive(ciphertexts, partial_key)
  File "/Users/clonsdale/Personal/manytime_interactive/manytime/interactive.py", line 191, in interactive
    Application(ciphertexts, Key(key)).run()
  File "/Users/clonsdale/Personal/manytime_interactive/manytime/interactive.py", line 143, in run
    self._loop.run()
  File "/Users/clonsdale/Personal/manytime_interactive/venv/lib/python3.6/site-packages/urwid/main_loop.py", line 286, in run
    self._run()
  File "/Users/clonsdale/Personal/manytime_interactive/venv/lib/python3.6/site-packages/urwid/main_loop.py", line 384, in _run
    self.event_loop.run()
  File "/Users/clonsdale/Personal/manytime_interactive/venv/lib/python3.6/site-packages/urwid/main_loop.py", line 788, in run
    self._loop()
  File "/Users/clonsdale/Personal/manytime_interactive/venv/lib/python3.6/site-packages/urwid/main_loop.py", line 816, in _loop
    self._entering_idle()
  File "/Users/clonsdale/Personal/manytime_interactive/venv/lib/python3.6/site-packages/urwid/main_loop.py", line 777, in _entering_idle
    callback()
  File "/Users/clonsdale/Personal/manytime_interactive/venv/lib/python3.6/site-packages/urwid/main_loop.py", line 572, in entering_idle
    self.draw_screen()
  File "/Users/clonsdale/Personal/manytime_interactive/venv/lib/python3.6/site-packages/urwid/main_loop.py", line 587, in draw_screen
    self.screen.draw_screen(self.screen_size, canvas)
  File "/Users/clonsdale/Personal/manytime_interactive/venv/lib/python3.6/site-packages/urwid/raw_display.py", line 804, in draw_screen
    assert cs in [None, "0", "U"], repr(cs)
AssertionError: b'0'
```

##### Affected versions (if applicable)
- [ ] `master` branch (specify commit)
- [x] Latest stable version from `pypi`
- [ ] Other (specify source)

##### Steps to reproduce (if applicable)
Spam capital A on the first line after running `mtp examples/sample.ciphertexts`

##### Expected/actual outcome
This should not crash urwid, perhaps b'0' needs to be added to the list on line 804 in draw_screen ?

--- a/urwid/util.py
+++ b/urwid/util.py
@@ -156,7 +156,7 @@ def apply_target_encoding( s ):
             sin = sl[0]
             assert isinstance(sin, bytes)
             sout.append(sin)
-            rle_append_modify(cout, (escape.DEC_TAG.encode('ascii'), len(sin)))
+            rle_append_modify(cout, (escape.DEC_TAG, len(sin)))
             continue
         sin, son = sl
         son = son.replace(SI, bytes())

